### DISCUSSION TITLE
syntax error at or near "RETURNING"

### DISCUSSION THREAD

**Discussion by app-o-matix** [sqlc-dev/sqlc#1255] [Q&A]
I'm getting this error when I run `sqlc generate`. In GoLand, it is highlighting the semicolon in `RETURNING *;`, after which my sql has a blank line then the magic comment for the next query. I remember seeing this before, but I don't recall what I did to overcome it. Any suggestions? Thanks!

**Comment by kuanbase** [comment]
I have got same error

**Comment by oliviernguyenquoc** [comment]
Didn't you forget a trailing comma in your request ?
